db/BaseService/index.js

```diff
@@ -0,0 +1,57 @@
+const assert = require("assert");
+const { knex } = require("../helpers");
+
+class BaseService {
```

Comment:
When I see service, I'm thinking of something more high level. Usual names for these kinds of class that handle database access are Dao or Repository.